fedora-builder/src/main/java/org/dataconservancy/nihms/builder/fs/ModelBuilder.java

```diff
@@ -255,35 +245,6 @@ DepositSubmission createDepositSubmission(Submission submissionEntity, HashMap<U
         User userEntity = (User)entities.get(submissionEntity.getUser());
         persons.add(createPerson(userEntity, DepositMetadata.PERSON_TYPE.submitter));
 
-        // Data from the Submission's Publication resource and its referenced Journal and Publisher resources
-        Publication publicationEntity = (Publication)entities.get(submissionEntity.getPublication());
-        manuscript.setTitle(publicationEntity.getTitle());
-        String doi = publicationEntity.getDoi();
-        try {
-            if (doi != null) {
-                doi = doi.trim();
-                article.setDoi(new URI(doi));
-            }
-        } catch (URISyntaxException e) {
-            String msg = String.format("Submission contained an invalid DOI URI: %s", doi);
-            throw new InvalidModel(msg, e);
-        }
-        // Available Publication data for which there is no place in the existing deposit model:
-        // Some of these properties are ignored because they are overwritten by the metadata, below.
-        // Leave commented out until this metadata is to be used.  Protect against NPEs.
-//        String publicationAbstract = publicationEntity.getPublicationAbstract();
-//        String pmid = publicationEntity.getPmid();
-//        String volume = publicationEntity.getVolume();
-//        String issue = publicationEntity.getIssue();
-
-        Journal journalEntity = (Journal)entities.get(publicationEntity.getJournal());
-        //journal.setJournalTitle(journalEntity.getName());
-        for (String issn : journalEntity.getIssns()) {
-            // Note: the current model only has room for one ISSN, but Fedora provides multiples
-            if (issn == journalEntity.getIssns().get(0)) {
-                //journal.setIssn(issn);
-            }
-        }
         // Is the model's ID the nlmta value?
```

Comment:
Since the journalEntity was removed above, this section (which uses it) should also be removed.